src/linker.rs

```diff
@@ -492,6 +492,20 @@ impl Linker {
         if !self.options.disable_expand_memcpy_in_order {
             args.push("--bpf-expand-memcpy-in-order".into());
         }
+        args.push(
+            format!(
+                "--inline-threshold={}",
```

Comment:
Have we verified that this works? And is it still necessary? This was copy pasted from rustc pre-NewPM, what does rustc do now?

Reply:
How does one verify? I don't know if it's necessary, this was just me being more conservative with what's changing.

Reply:
LLVMParseCommandLineOptions exits on failure to parse, so I'm pretty sure it works.

Reply:
The old values were set to match the rustc values, so first we should check what values (if any) rustc uses with the new PM. If we do need to override them, we should check that the inline pass actually does pick up this option, which is something I've done in the past... by adding prints in the passes. Or like setting inline threshold to 0 and making sure that nothing is inlined, to 10000 and making sure things are inlined

Reply:
https://github.com/rust-lang/rust/issues/89742 -- looks like rust doesn't do this anymore. I'll remove it.

Reply:
Done.